### PR TITLE
Makes robot repair automatically select next damaged limb

### PR DESCRIPTION
**Pull request by Walarks**
## About The Pull Request

Robot repair with cable coil and welder automatically selects next damaged limb.

Also kind of normalizes some of the repair code.
## Why It's Good For The Game

Less monotonous clicking.
## Changelog
:cl:
qol: robot repair automatically starts repairing the next damaged limb
/:cl:

### CODE REVIEW

**Comment by Losenis** [comment]
Based

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -483,10 +483,11 @@ GLOBAL_LIST(cable_radial_layer_list)
...
 
 	var/datum/limb/affecting = user.client.prefs.toggles_gameplay & RADIAL_MEDICAL ? radial_medical(H, user) : H.get_limb(user.zone_selected)
+
 	if(!affecting)
 		return TRUE
 
-	if(affecting.limb_status != LIMB_ROBOT)
+	if(!(affecting.limb_status & LIMB_ROBOT))
 		balloon_alert(user, "Limb not robotic")
 		return TRUE
```
Comment (TiviPlus) [inline code/modules/power/cable.dm:492]: Im suprised this isnt conflicting

**Review by TiviPlus — APPROVED** [review]

**Comment by TiviPlus** [comment]
> Based

Based on what

**Comment by Losenis** [comment]
> Based on what

https://www.youtube.com/watch?v=LrNu-SuFF_o

**Pull request merged**